featuretools/entityset/serialize.py

```diff
@@ -92,6 +98,9 @@ def write_entity_data(entity, path, format='csv', **kwargs):
     df = entity.df
 
     if format == 'csv':
+        df = df.copy()
```

Comment:
Is this something we only need to run with a koalas dataframe?

Reply:
Yep, I'll pull it into an if block